packages/create-cloudflare/src/frameworks/solid/templates.ts

```diff
@@ -1,9 +1,12 @@
 export const viteConfig = `
-import solid from "solid-start/vite";
-import { defineConfig } from "vite";
-import cloudflare from "solid-start-cloudflare-pages";
+import { defineConfig } from "@solidjs/start/config";
 
 export default defineConfig({
-  plugins: [solid({ adapter: cloudflare({}) })],
+    // @ts-ignore
```

Comment:
the `// @ts-ignore` is needed so that we don't have to put a big `solid` object here

hopefully this should not be necessary later but for now I guess this could be ok

see: https://github.com/solidjs/solid-start/issues/1217#issuecomment-1879638879